helpdesk_scope/models/helpdesk_scope.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class HelpdeskScope(models.Model):
+    _name='helpdesk.scope'
+
+    helpdesk_scope = fields.Char(string="Scope")
+    description = fields.Text(string="Description")
```

Comment:
@osi-scampbell 
* No need to import api
* You need to add `_description = 'Helpdesk Scope'`
* You need to have a `name` field. Please refer to the design.